README.md

```diff
@@ -58,6 +58,7 @@ This repository contains examples of various algorithms written on different pro
 4. Commit your changes: `git commit -am 'Add some feature'`.
 
    Go through [Commit Messages guidelines](CONTRIBUTING.md#write-good-commit-messages)
+
 5. Push to the branch: `git push origin my-new-feature`
 6. Submit a pull request :smile:
 
```

Comment:
The changes in this file is redundant. Kindly change only line that is required.